src/views/Accessibility/AccessibiltyRequest/Documents/New/index.tsx

```diff
@@ -125,49 +159,179 @@ const New = () => {
+                            error={!!flatErrors['documentType.commonType']}
+                          >
+                            <fieldset className="usa-fieldset margin-top-4">
+                              <legend className="usa-label margin-bottom-1">
+                                What type of document are you uploading?
+                              </legend>
+                              <FieldErrorMsg>
+                                {flatErrors['documentType.commonType']}
+                              </FieldErrorMsg>
+                              {([
+                                'AWARDED_VPAT',
+                                'TEST_PLAN',
+                                'TESTING_VPAT',
+                                'TEST_RESULTS',
+                                'REMEDIATION_PLAN'
+                              ] as AccessibilityRequestDocumentCommonType[]).map(
+                                commonType => {
+                                  return (
+                                    <Field
+                                      key={`FileUpload-CommonType${commonType}`}
+                                      as={RadioField}
+                                      checked={
+                                        values.documentType.commonType ===
+                                        commonType
+                                      }
+                                      id={`FileUpload-CommonType${commonType}`}
+                                      name="documentType.commonType"
+                                      label={getDocumentTypeLabel(commonType)}
+                                      onChange={() => {
+                                        setFieldValue(
+                                          'documentType.commonType',
+                                          commonType
+                                        );
+                                        setFieldValue(
+                                          'documentType.otherType',
+                                          ''
+                                        );
+                                      }}
+                                      value={commonType}
+                                    />
+                                  );
+                                }
+                              )}
+                              <Field
+                                as={RadioField}
+                                checked={
+                                  values.documentType.commonType === 'OTHER'
+                                }
+                                id="FileUpload-CommonTypeOTHER"
+                                name="documentType.commonType"
+                                label={getDocumentTypeLabel(
+                                  AccessibilityRequestDocumentCommonType.OTHER
+                                )}
+                                onChange={() => {
```

Comment:
I don't think we need to explicitly set an `onChange`. `setFieldValue('documentType.commonType', 'OTHER')` is the default behavior that comes with `Field`